samples/host_verify/README.md

```diff
@@ -0,0 +1,68 @@
+
+# The host verification sample
+
+- Demonstrates the attestation of a remote host
+- Requires only the `openenclave/host_verify.h` header, as well as some C standard header files
+- Also runs in simulation mode and when both BUILD_ENCLAVES and HAS_QUOTE_PROVIDER flags are turned off
+
+Prerequisites: you may want to read [Common Sample Information](../README.md#common-sample-information) before going further
+
+## About the host verification sample
+
+When a host is doing host verification, or remote host attestation to be exact, it means that it is trying to authenticate its hardware and software settings so that it can determine whether or not to trust the remote host.
```

Comment:
```suggestion
When an application is doing host-side enclave verification, it means that a host application is trying to authenticate a remote enclave's hardware and software settings so that the application can determine whether or not to trust the remote enclave.
```

Reply:
Thanks, this description made me understand the concept better too!